### DISCUSSION TITLE
Add meta field for 'original source' such as magazine/cookbook for from-print

### DISCUSSION THREAD

**Discussion by michaelkrieger** [mealie-recipes/mealie#972] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Record a reference other than the 'original URL', such as the cookbook or magazine the recipe was typed/created from.
For example, a cookbook only recipe I just entered and in the description I included "From: Ina Garten's Modern Comfort Food (Pg. 114)" or "From: Cooks Illustrated, December 2021".

**(Optional): Suggest A Solution**
* A simple text field such as "Original Source" would work.
  * Using this method, it could be prominently displayed with the description
* Alternately, reuse the alternate URL field and detect if it is a URL or plain text... though this may impact more if other items depend on that being a URL and don't check if it is.

**Additional Information**
- [X] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
You can utilize notes (or description) to achieve this; I'm not sure I see a reason to create a specific field for it
